examples/two-columns-layout/style.css

```diff
@@ -60,9 +60,162 @@ body {
   transition: background 0.2s ease-out;
 }
 
+.aa-Item:hover .aa-ItemPicture img {
+  transform: scale(1.1);
+}
+
+.aa-ItemPicture {
+  width: 100%;
+  overflow: hidden;
+}
+
+.aa-ItemPicture img {
+  object-fit: cover;
+  width: 100%;
+  height: auto;
+  border-radius: 3px;
+  transition: transform 1.8s ease-out;
+  transform-origin: center bottom;
+}
+
+/* Products */
+.aa-PanelSection--products {
+  padding: var(--aa-spacing-half);
+  transition: background 0.2s ease-out;
+}
+
+.aa-PanelSection--products-preview {
+  background: rgba(
+    var(--aa-selected-color-rgb),
+    var(--aa-selected-color-alpha)
+  );
+  border-radius: 3px;
+}
+
+.aa-PanelSection--products .aa-List {
+  display: flex;
+  column-gap: var(--aa-spacing);
+}
+
+.aa-PanelSection--products .aa-Item {
+  align-items: flex-start;
+  width: 100%;
+}
+
+.aa-PanelSection--products .aa-Item:hover .aa-ItemFavorite {
+  opacity: 1;
+}
+
+.aa-ProductItem {
+  height: 100%;
+  min-height: 380px;
+  align-items: flex-start;
+}
+
+.aa-ProductItem .aa-ItemPicture {
+  position: relative;
+}
+
+.aa-ProductItem .aa-ItemContentBody {
+  width: 100%;
+  display: flex;
+  flex-grow: 1;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.aa-ProductItem .aa-ItemPictureBlurred {
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  transition: opacity 0.2s ease-out, visibility 0s linear 0.2s;
+}
+
+.aa-ProductItem .aa-ItemPictureBlurred--loaded {
+  opacity: 0;
+  visibility: hidden;
+}
+
+.aa-ProductItem .aa-ItemContent {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+}
+
+.aa-ProductItem .aa-ItemContentBrand {
+  font-size: 0.7em;
+  text-transform: uppercase;
+  color: rgb(var(--aa-muted-color-rgb));
+}
+
+.aa-ProductItem .aa-ItemContentTitle {
+  white-space: normal;
+  font-size: 0.9em;
+}
+
+.aa-ProductItem .aa-ItemContentPriceCurrent {
+  font-weight: bold;
+}
+
+.aa-ProductItem .aa-ItemContentPriceDiscounted {
+  font-size: 0.9em;
+  text-decoration: line-through;
+  color: rgb(var(--aa-muted-color-rgb));
+}
+
+.aa-ProductItem .aa-ItemContentPrice {
+  display: flex;
+  column-gap: var(--aa-spacing-half);
+}
+
+.aa-ProductItem .aa-ItemContentRating ul {
+  display: flex;
+  list-style: none;
+  padding: 0;
+}
+
+.aa-ProductItem .aa-ItemContentRating {
+  display: flex;
+  align-items: center;
+  column-gap: 0.4em;
+  margin-top: var(--aa-spacing-half);
+}
+
+.aa-ProductItem .aa-StarIcon {
+  width: 1.3em;
+  height: 1.3em;
+  color: rgb(var(--aa-primary-color-rgb));
+}
+
+.aa-ProductItem .aa-ItemContentRatingReviews {
+  font-size: 0.8em;
+}
+
+.aa-ProductItem .aa-ItemFavorite {
+  z-index: 5;
+  position: absolute;
+  right: 4px;
+  top: 4px;
+  background-color: #fff;
+  border-radius: 3px;
+  box-shadow: 0px 4px 8px rgba(35, 38, 59, 0.15);
+  cursor: pointer;
+  opacity: 0;
```

Comment:
Why is it hidden on desktop?

Also, we should use the `hidden` HTML attribute instead since we don't use the `display` property.

Reply:
On desktop, it is only visible when hovering an item. And on mobile, it is always visible (no hover state).